test/e2e/framework/ingress/ingress_utils.go

```diff
@@ -106,6 +106,14 @@ const (
 	// names of GCP resources such as forwarding rules, url maps, target proxies, etc
 	// that it created for the corresponding ingress.
 	StatusPrefix = "ingress.kubernetes.io"
+
+	// poll is how often to Poll pods, nodes and claims.
+	poll = 2 * time.Second
```

Comment:
Is there a better place for these const 🤔 
They are also used by some other methods in e2e/framework/node, and I just temporarily duplicate them.

Reply:
I would recommend to give the const a more specific name for this package.
There isn't a lot more that we can do with it.

Reply:
These values will not be exported, so I think the names are fine.